domain/src/main/java/com/harman/domain/repository/RandomNumberRepository.kt

```diff
@@ -0,0 +1,4 @@
+package com.harman.domain.repository
+
+class RundomNumberRepository {
```

Comment:
Please put this class into data layer, and add an interface for this class here (in domain layer)